pkg/dnfjson/dnfjson.go

```diff
@@ -160,6 +163,10 @@ func (s *Solver) GetCacheDir() string {
 	return filepath.Join(s.cache.root, b)
 }
 
+func (s *Solver) SetProxy(proxy string) {
```

Comment:
should this validate the proxy string? Make sure it's a valid URL or not?

Does it need some doc comment above?

Reply:
We do have URL validation in some places: https://github.com/osbuild/images/blob/d1d5119e1b98d1d365f0a91e168384f7f59e78b4/pkg/ostree/ostree.go#L111-L113

That's pretty simple and should be enough.

A comment would be good too.

Reply:
will do